GitTfs/Commands/Init.cs

```diff
@@ -102,6 +102,13 @@ private void DoGitInitDb()
             }
             globals.Repository = gitHelper.MakeRepository(globals.GitDir);
 
+            if (string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.name"))
+                && string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.email")))
+            {
+                globals.Repository.SetConfig("user.name", "git-tfs");
+                globals.Repository.SetConfig("user.email", "placeholder@git-tfs.com");
+            }
```

Comment:
Could `CI.proj` or the integration tests run a setup step that sets `user.name` and `user.email`?